lib/Core/Executor.cpp

```diff
@@ -2998,12 +2977,33 @@ void Executor::run(ExecutionState &initialState) {
     }
 
     klee_message("seeding done (%d states remain)", (int) states.size());
+    klee_message("<CONC> sizeof prio2: %lu", prio2.size());
+    klee_message("<CONC> sizeof prio1: %lu", prio1.size());
+    klee_message("<CONC> sizeof prio0: %lu", prio0.size());
+    for ( auto &s : prio2 ) {
+        //klee_message("Solving prio2");
+        interpreterHandler->processTestCase(*s, 0, 0); // TODO: maybe create our own terminate function
+        interpreterHandler->incPathsExplored(); // not sure this is correct
+    }
+    for ( auto &s : prio1 ) {
+        interpreterHandler->processTestCase(*s, 0, 0); // TODO: maybe create our own terminate function
+        interpreterHandler->incPathsExplored(); // not sure this is correct
+    }
+    /*for ( auto &s : prio0 ) {
+        klee_message("Solving prio0");
+        interpreterHandler->processTestCase(*s, 0, 0); // TODO: maybe create our own terminate function
+        interpreterHandler->incPathsExplored(); // not sure this is correct
+    }*/
+
+    klee_message("states size() after: %lu\n", states.size());
 
     if (OnlySeed) {
```

Comment:
are we passing --only-seed when running KLEE?

Reply:
Yes, currently I run it like so:
/path/to/klee/build/bin/klee --libc=uclibc --posix-runtime -server-seed-dir=/root/git/test/inotify_test/watch -sync-dir=/root/git/test/syncdir --named-seed-matching --max-solver-time=5 -only-replay-seeds -only-seed -allow-seed-extension -allow-seed-truncation -allocate-determ=true -solver-backend=z3 /path/to/mytest.bc --sym-stdin 1000 &> dump

Reply:
ok. So we should check we're not dumping the same state twice (see my https://github.com/leonbett/klee/pull/1#discussion_r450565108). To speed things up (after you've debugged), you can also use `-only-output-states-covering-new`. But your code should also work without this option, so let's not use it yet - it's just an optimization.

Reply:
BTW, what is the `sync-dir` option for? I have not used it myself.

Reply:
Thanks for the info!
sync-dir is an option I've added (https://github.com/leonbett/klee/blob/debug2/tools/klee/main.cpp#L140). It's the location where raw test cases are output for a fuzzer to pick them up.

Reply:
ok got it! Maybe we should call these `--stateful-sync-indir` and `--stateful-sync-outdir`. It's not important for now.